pkgs/applications/networking/cluster/kpt/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "kpt";
-  version = "0.39.3";
+  version = "1.0.0-beta.49";
 
   src = fetchFromGitHub {
-    owner = "GoogleContainerTools";
+    owner = "kptdev";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-vidrKfmP0Lw6EYYufLDxh3ROOJ3hPIusDTI/Hr73NYM=";
+    hash = "sha256-whqA1ACN7vXSDWVi55/fJKnuBrDCIdOpyDS8KNrhZq0=";
```

Source hash for `1.0.0-beta.55`

```suggestion
    hash = "sha256-MVrJUsMpt1L56ezy2b2G6Aac9kpe4QgfSosR+PeTuBQ=";
```